src/proj.h

```diff
@@ -600,6 +600,19 @@ size_t PROJ_DLL proj_trans_generic (
     double *t, size_t st, size_t nt
 );
 
+int PROJ_DLL proj_trans_bounds(
```

Comment:
reference to add to it in docs/source/development/reference/functions.rst using a .. doxygenfunction:: style of inclusion at the appropriate place